utils/products.js

```diff
@@ -1,152 +1,676 @@
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger-backend Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "managed-simple-data-exchanger-frontend",
+    productDescription: "TBD",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/managed-simple-data-exchanger-frontend",
+    ],
+    committers: [
+      "https://github.com/almadigabor",
+      "https://github.com/tomaszbarwicki",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request managed-simple-data-exchanger-frontend Team",
+    hasBoard: false,
+    showVersion: true,
+  },
+  {
+    productName: "Policy Hub",
+    productDescription:
+      "The Policy Hub enables data providers, consumers and app providers to access a single-point-of-truth for Catena-X policies, attributes and templates for policy rules.",
+    githubRepo: ["https://github.com/eclipse-tractusx/policy-hub"],
+    committers: [
+      "https://github.com/evegufy",
+      "https://github.com/Phil91",
+      "https://github.com/tom-rm-meyer-ISST",
+    ],
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Tractus-X Policy Hub Team",
+    hasBoard: false,
+    showVersion: true,
   },
-
   {
     productName: "Portal & Marketplaces",
-    productDescription: "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
+    productDescription:
+      "The Portal facilitates the operations for dataspace members (companies), it includes functionalities such as registration, technical onboarding and marketplaces.",
     githubRepo: [
       "https://github.com/eclipse-tractusx/portal",
       "https://github.com/eclipse-tractusx/portal-backend",
       "https://github.com/eclipse-tractusx/portal-frontend",
       "https://github.com/eclipse-tractusx/portal-frontend-registration",
       "https://github.com/eclipse-tractusx/portal-assets",
-      "https://github.com/eclipse-tractusx/portal-iam"
+      "https://github.com/eclipse-tractusx/portal-iam",
+      "https://github.com/eclipse-tractusx/portal-shared-components",
     ],
     committers: [
       "https://github.com/evegufy",
       "https://github.com/ntruchsess",
       "https://github.com/oyo",
-      "https://github.com/Phil91"
+      "https://github.com/Phil91",
     ],
-    mailTo: "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
+    mailTo:
+      "tractusx-dev@eclipse.org?subject=Request Portal and Marketplaces Team",
     hasBoard: false,
-    showVersion: true
+    showVersion: true,
+  },
+  {
+    productName: "Predictive Unit Real-Time Information Service",
+    productDescription:
+      "Predictive Unit Real-Time Information Service (PURIS) for Short Term Demand and Capacity Management",
+    githubRepo: [
+      "https://github.com/eclipse-tractusx/puris",
+      "https://github.com/eclipse-tractusx/puris-backend",
```

Comment:
Please remove the backend and frontend repository as those are archived.

From Committers, please remove @tomaszbarwicki. @mhellmeier for you it's OK to be listed here, isn't it?